package.json

```
@@ -59,7 +59,7 @@
     "start": "ffbinaries ffmpeg ffprobe -o=binaries && rescripts start",
     "build": "tsc && ffbinaries ffmpeg ffprobe -o=binaries && rescripts build",
     "test": "tsc --project tsconfig.spec.json && ava **/*.spec.js",
-    "integration": "tsc --project tsconfig.integration.json && ava **/*.integration.js",
+    "integration": "tsc --project tsconfig.integration.json && ffbinaries ffmpeg ffprobe -o=binaries && ava **/*.integration.js",
```

Since the ffbinaries line command is copied in three places in our npm scripts, now seems like a good time to make this its own script "ff"
```suggestion
    "integration": "tsc --project tsconfig.integration.json && npm run ff && ava **/*.integration.js",
```